boot.js

```diff
@@ -402,8 +402,8 @@ function thenify () {
   }
 }
 
-function callWithCbOrNextTick (func, cb, context) {
-  context = this._server
+function callWithCbOrNextTick (func, cb) {
+  const context = this._server
```

Comment:
this is never called with the 3rd argument?

Reply:
never called with 3rd argument